common/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -112,9 +112,7 @@ public MasterClient(InetSocketAddress masterAddress, ExecutorService executorSer
    * @param length
    * @param checkpointPath
    * @return true if checkpoint is added for the <code>fileId</code> and false otherwise
-   * @throws FileDoesNotExistException
-   * @throws SuspectedFileSizeException
-   * @throws BlockInfoException
+   * @throws IOException
```

Comment:
Could you add the info for the exceptions after the throws clause? Ie. `@throws IOException if the file does not exist, has the wrong size, or its block information is corrupted.`